splash/tests/test_render.py

```diff
@@ -627,6 +626,38 @@ def _iframes_request(self, params):
         return self.request(query).json()
 
 
+class CommandLineOptionsTest(BaseRenderTest):
+
+    def test_max_timeout(self):
+        with SplashServer(extra_args=['--max-timeout=5.0']) as splash:
+            r1 = requests.get(
+                url=splash.url("render.html"),
+                params={
+                    'url': self.mockurl("delay?n=10"),
+                    'timeout': '30.0',
+                },
+            )
+            self.assertStatusCode(r1, 400)
+
+            r2 = requests.get(
+                url=splash.url("render.html"),
+                params={
+                    'url': self.mockurl("delay?n=10"),
+                    'timeout': '5.0',
```

Comment:
we should also check a case when 'timeout' argument is not passed: loading should fail if it takes longer than --max-timeout